cpp/src/text/subword/data_normalizer.cu

```diff
@@ -296,20 +297,22 @@ uvector_pair data_normalizer::normalize(char const* d_strings,
                       auto const offset = d_offsets[0];  // adjust for any offset to the offsets
                       return d_offsets[idx] - offset;
                     });
-  auto const bytes_count = d_strings_offsets->element(num_strings, stream);
+  auto const bytes_count = d_strings_offsets->element(input.size(), stream);
   if (bytes_count == 0) {  // if no bytes, nothing to do
     return uvector_pair{std::make_unique<rmm::device_uvector<uint32_t>>(0, stream),
-                        std::make_unique<rmm::device_uvector<cudf::size_type>>(0, stream)};
+                        std::make_unique<rmm::device_uvector<int64_t>>(0, stream)};
   }
 
-  cudf::detail::grid_1d const grid{bytes_count, THREADS_PER_BLOCK, 1};
-  size_t const threads_on_device  = grid.num_threads_per_block * grid.num_blocks;
+  int64_t const threads_per_block = THREADS_PER_BLOCK;
```

Comment:
Why is this one `int64_t` and not `size_t`?

Reply:
Because the `cudf::util::div_rounding_up_safe` requires both of its parameters to be the same type.